receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -402,15 +402,15 @@ metrics:
   k8s.resource_quota.hard_limit:
     enabled: true
     description: The upper limit for a particular resource in a specific namespace. Will only be sent if a quota is specified. CPU requests/limits will be sent as millicores
-    unit: 1
+    unit: ""
```

Comment:
This is a weird one. I think it would be more appropriate to have a unit like `{resource}`  which imo is more representative of the quantity having a unit depending on the resource type (e.g. resource = cpu, the unit is cores).